src/core/Akka.TestKit/TestKitBase_ExpectAllWithPredicates.cs

```diff
@@ -78,7 +78,7 @@ await foreach (var item in enumerable)
 
         var unexpectedMessages = new List<object>();
         var receivedMessages = InternalReceiveNAsync(predicateInfos.Count, max, shouldLog, cancellationToken)
-            .ConfigureAwait(false).WithCancellation(cancellationToken);
+            .WithCancellation(cancellationToken);
```

Comment:
We don't need `.WithCancellation()` here, token is already passed into the method.